package.json

```diff
@@ -20,7 +20,6 @@
     "release:major": "./publish.sh major",
     "clean": "rimraf dist coverage",
     "reinstall": "rimraf flow-typed && rimraf node_modules && npm install && flow-typed install",
-    "postinstall": "flow-typed install",
```

Comment:
This will actually run the `flow-typed` command when all consumers install your module, which is unnecessary as they'll just be consuming your `brainblocks.js` module.

Reply:
Agreed on this change. I'd already made the fix locally, I'm pushing it up as we speak, but thanks for the PR!